railties/test/generators/generators_test_helper.rb

```diff
@@ -122,6 +122,12 @@ def assert_compose_file
     end
   end
 
+  def assert_devcontainer_json_file
```

Comment:
Note to self: Use this helper in the other tests. But doing so here will create conflicts with https://github.com/rails/rails/pull/51481 so I will do it in that PR or a future PR.